### PR TITLE
[DGPFET-2] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by olaide-hok**

### CODE REVIEW

File: submissions/habeebKareem/html/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```
Comment (TundeOladejo) [inline submissions/habeebKareem/html/index.html:1]: Please format this file properly

File: tests/package.json [hunk 2]
```diff
@@ -4,6 +4,6 @@
     "chai": "^4.3.4",
```
Comment (TundeOladejo) [inline tests/package.json:4]: Please kindly remove your changes. It has already been updated on the master branch.

Comment (TundeOladejo) [reply tests/package.json:4]: Also, delete the `chromedriver.exe` file you added below, and please refrain from **modifying** any code in the `tests` folder. Thank you.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Comment by ToluEnny24** [comment]
@olaide-hok  I will be closing this Pull Request so you can effect the necessary changes. Pull the update from the master branch, add the logfile, resolve the conflicts by opening in vscode and accepting the changes from master.
Do not make any changes to the tests scripts. 
Hint: try to copy your html task to another location before pulling from master.

**Pull request closed without merging**